acceptance_tests/features/steps/events_emitted.py

```diff
@@ -23,33 +23,35 @@ def case_updated_msg_sent_with_values(context, case_field, expected_field_value)
                             'The updated case field must match the expected value')
 
 
-@step("UAC_UPDATED messages are emitted with active set to true")
-def check_uac_updated_msgs_emitted_with_qid_active(context):
-    context.emitted_uacs = get_uac_updated_events(len(context.emitted_cases))
+@step("UAC_UPDATED messages are emitted with active set to {active_str}")
+def check_uac_updated_msgs_emitted_with_qid_active(context, active_str):
+    context.emitted_uacs = get_uac_updated_events(context.collex_id, len(context.emitted_cases))
     _check_uacs_updated_match_cases(context.emitted_uacs, context.emitted_cases)
-    _check_uac_updated_events(context.emitted_uacs, context.collex_id)
 
+    active = False
+    if active_str == 'true':
+        active = True
```

Comment:
Worth using `strtobool` to parse this to be safe, as is you could write `"True"` in the step and it would expect `active = False`

Reply:
As discussed, we'll use this to define boolean type from steps: https://jenisys.github.io/behave.example/tutorials/tutorial10.html